test/test_intersect.py

```diff
@@ -87,18 +89,19 @@ def test_cube_intersect():
     assert np.isclose(proc.returncode, 0)
 
     ## hard code what it should be and check it matches
-    wm_file = os.path.join(SCENARIO_DIR, f'{WM}_hydro_{date}T{time.replace(":", "")}_ztd.tiff')
-    da      = xr.open_dataset(wm_file)['band_data']
-    gold    = {'GMAO': 2.045914, 'ERA5': 2.061974, 'HRRR': 3.0972726}
+    WM_file = os.path.join(SCENARIO_DIR, f'{WM}_hydro_{date}T{time.replace(":", "")}_ztd.tiff')
+    da      = xr.open_dataset(WM_file)['band_data']
+    gold    = {'GMAO': 2.0541468, 'ERA5': 2.0696816}#, 'HRRR': 3.0972726}
```

Comment:
we now produce data on a cube of different shape; these are the mean values so are updated accordingly

Reply:
Can you elaborate more. Yes the cube and buffering has changed, are you not computing these values for a fixed lon lat? In that case it would not be impacted by the buffering.

Reply:
We should take a single value, can be seperate PR.